### ISSUE TITLE
Remove glossary or symbols

### ISSUE THREAD

**Issue by jaasilva** [joaomlourenco/novathesis#171]
I'm using version 6.4.0 (local LaTeX installation).

Currently, I'm not interested in keeping the glossary nor the symbols list.
However, in file '6_files.tex', when I try to comment out any of these lines:
```
%\ntaddfile{glossaries}[glossary]{glossary}
\ntaddfile{glossaries}[acronyms]{acronyms}
%\ntaddfile{glossaries}[symbols]{symbols}
```
I receive an error when compiling:
`Missing endcsname inserted. \begin{document}`

For this I haven't found a solution yet.

**Comment by petersaints** [comment]
@jaasilva Check this report I made sometime ago: https://github.com/joaomlourenco/novathesis/issues/166

With an older TeX Live packaged with Ubuntu 20.10 I had the same error. Once I installed the latest TeX Live from the official website I had no such problem. It also worked well under Windows with an updated MiKTeX installation.

**Comment by jaasilva** [comment]
@petersaints thanks for the pointer.
I'm using Ubuntu. I'll try to update the TeX Live installation.

**Comment by joaomlourenco** [comment]
@jaasilva, did you have success?  Do you confirm that updating the TL installation it solves the problem?

**Comment by petersaints** [comment]
@joaomlourenco The problem returned with the last update I made to my MiKTeX install on Windows. I haven't tested on Linux with an updated TeX Live install yet.

**Comment by joaomlourenco** [comment]
Confirmed!  I'll check what's happening.

**Comment by joaomlourenco** [comment]
Fixed in version 6.4.2.
